swig/include/python/gdal_python.i

```diff
@@ -3806,6 +3806,202 @@ def BuildVRT(destName, srcDSOrSrcDSTab, **kwargs):
         return BuildVRTInternalNames(destName, srcDSNamesTab, opts, callback, callback_data)
 
 
+def TileIndexOptions(options=None,
+                     overwrite=None,
+                     recursive=None,
+                     filenameFilter=None,
+                     minPixelSize=None,
+                     maxPixelSize=None,
+                     format=None,
+                     layerName=None,
+                     locationFieldName="location",
+                     outputSRS=None,
+                     writeAbsolutePath=None,
+                     skipDifferentProjection=None,
+                     gtiFilename=None,
+                     xRes=None,
+                     yRes=None,
+                     outputBounds=None,
+                     colorInterpretation=None,
+                     noData=None,
+                     bandCount=None,
+                     mask=None,
+                     metadataOptions=None,
+                     fetchMD=None):
+    """Create a TileIndexOptions() object that can be passed to gdal.TileIndex()
+
+    Parameters
+    ----------
+    options:
+        can be be an array of strings, a string or let empty and filled from other keywords.
+    overwrite:
+        Whether to overwrite the existing tile index
+    recursive:
+        Whether directories specified in source filenames should be explored recursively
+    filenameFilter:
+        Pattern that the filenames contained in directories pointed by <file_or_dir> should follow. '*' and '?' wildcard can be used. String or list of strings.
+    minPixelSize:
```

Comment:
```suggestion
    minPixelcount:
```

Reply:
It may mean size even here and later as well.